### DISCUSSION TITLE
initial conditions

### DISCUSSION THREAD

**Discussion by wowodejiajia** [idaholab/moose#26499] [Q&A General] [answered]
Dear MOOSE experts,

I have finished calculating a task, and I want to calculate another task using the results of this task as initial conditions. What should I do? How do I modify the original input file?
[input file.docx](https://github.com/idaholab/moose/files/13856042/input.file.docx)

Thank you.

**Comment by GiudGiud** [comment, marked as answer]
Hello

Use either of these techniques:
https://mooseframework.inl.gov/application_usage/restart_recover.html
more likely you will want to use a Checkpoint restart as it is the safest / most widely applicable one. For a simple case, you may be able to restart from Exodus.

Guillaume

**Reply by wowodejiajia** [reply]
Okay, thank you.
